meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.8'
+  version: '0.1.9'
 
 source:
   git_url: https://github.com/dgursoy/tomopy.git
```

Comment:
If you are targeting a git repo for a fixed release you should also include a `git_rev` entry other wise if you re-run old recipes you will get the new code version with the old number.